### PR TITLE
Update smp v2 notebook examples to not require users to move scripts.

### PR DESCRIPTION
**Pull request by viclzhu**
*Description of changes:*
     * Update smp v2 notebooks to be more consistent with each other.

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x] I have read the [CONTRIBUTING](https://github.com/aws/amazon-sagemaker-examples/blob/master/CONTRIBUTING.md) doc and adhered to the example notebook best practices
- [x] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-sagemaker-examples/blob/master/README.md)
- [x] I have tested my notebook(s) and ensured it runs end-to-end
- [x] I have linted my notebook(s) and code using `black-nb -l 100 {path}/{notebook-name}.ipynb`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: training/distributed_training/pytorch/model_parallel_v2/gpt-neox/smp-finetuning-gpt-neox-fsdp-tp.ipynb [hunk 1]
```diff
@@ -833,17 +837,6 @@
...
   },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "# Use the private preview image from ECR which contains all\n",
-    "# libraries and dependencies required for our SageMaker job.\n",
-    "# image = CONTAINER_IMAGE"
```
Comment (sheilaliuxl) [inline training/distributed_training/pytorch/model_parallel_v2/gpt-neox/smp-finetuning-gpt-neox-fsdp-tp.ipynb:844]: nit: Can remove def for `CONTAINER_IMAGE` at the top.

File: training/distributed_training/pytorch/model_parallel_v2/gpt-neox/smp-train-gpt-neox-fsdp-tp.ipynb [hunk 2]
```diff
@@ -865,7 +867,6 @@
     "        },\n",
     "    },\n",
     "    framework_version=\"2.0.1\",\n",
-    "    # image_uri=$IMAGE,  # Either provide `framework_version` or `image_uri`.\n",
```
Comment (sheilaliuxl) [inline training/distributed_training/pytorch/model_parallel_v2/gpt-neox/smp-train-gpt-neox-fsdp-tp.ipynb:868]: This is intentional, a placeholder for users to BYOC.

File: training/distributed_training/pytorch/model_parallel_v2/gpt-neox/smp-train-gpt-neox-fsdp-tp.ipynb [hunk 3]
```diff
@@ -947,7 +948,6 @@
     "            },\n",
     "        },\n",
     "        framework_version=\"2.0.1\",\n",
-    "        # image_uri=$IMAGE,  # Either provide `framework_version` or `image_uri`.\n",
```
Comment (sheilaliuxl) [inline training/distributed_training/pytorch/model_parallel_v2/gpt-neox/smp-train-gpt-neox-fsdp-tp.ipynb:950]: ditto

File: training/distributed_training/pytorch/model_parallel_v2/llama_v2/smp-finetuning-llama-fsdp-tp.ipynb [hunk 4]
```diff
@@ -818,17 +822,6 @@
...
   },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "# Use the private preview image from ECR which contains all\n",
-    "# libraries and dependencies required for our SageMaker job.\n",
-    "# image = CONTAINER_IMAGE"
```
Comment (sheilaliuxl) [inline training/distributed_training/pytorch/model_parallel_v2/llama_v2/smp-finetuning-llama-fsdp-tp.ipynb:829]: ditto to remove def for `CONTAINER_IMAGE`

File: training/distributed_training/pytorch/model_parallel_v2/gpt-neox/smp-finetuning-gpt-neox-fsdp-tp.ipynb [hunk 5]
```diff
@@ -859,7 +852,7 @@
     "smp_estimator = PyTorch(\n",
     "    entry_point=\"train.py\",\n",
     "    hyperparameters=hyperparameters,\n",
-    "    source_dir=os.getcwd(),\n",
+    "    source_dir=os.path.join(os.getcwd(), \"../shared-scripts\"),\n",
```
Comment (sheilaliuxl) [inline training/distributed_training/pytorch/model_parallel_v2/gpt-neox/smp-finetuning-gpt-neox-fsdp-tp.ipynb:855]: Thanks for the change, I assume that you have tried this E2E.

File: training/distributed_training/pytorch/model_parallel_v2/gpt-neox/smp-finetuning-gpt-neox-fsdp-tp.ipynb (hunk 1 above)
Comment (viclzhu) [reply training/distributed_training/pytorch/model_parallel_v2/gpt-neox/smp-finetuning-gpt-neox-fsdp-tp.ipynb:844]: Done!

File: training/distributed_training/pytorch/model_parallel_v2/gpt-neox/smp-finetuning-gpt-neox-fsdp-tp.ipynb (hunk 5 above)
Comment (viclzhu) [reply training/distributed_training/pytorch/model_parallel_v2/gpt-neox/smp-finetuning-gpt-neox-fsdp-tp.ipynb:854]: Yes, I tested on one notebook E2E which worked, so assuming will work for all the notebooks.

**Comment by viclzhu** [comment]
Thanks for the review! Updated PR with changes

**Comment by sheilaliuxl** [comment]
I think your 2nd commit msg need some rephrase: "Reformat notebooks with black-nb."

Otherwise LGTM.

**Review by sheilaliuxl — APPROVED** [review]

**Comment by viclzhu** [comment]
Updated thanks!

**Review by mchoi8739 — DISMISSED** [review]

**Review by sthodimi — APPROVED** [review]
LGTM

**Pull request merged**